components/fonts/platform/freetype/ohos/font_list.rs

```diff
@@ -238,4 +228,6 @@ pub fn fallback_font_families(options: FallbackFontSelectionOptions) -> Vec<&'st
     families
 }
 
-pub static SANS_SERIF_FONT_FAMILY: &'static str = "HarmonyOS Sans";
+pub fn default_system_generic_font_family(_generic: GenericFontFamily) -> LowercaseFontFamilyName {
+    "Harmony OS Sans".into()
```

Comment:
```suggestion
    "HarmonyOS Sans".into()
```

Adding the space before the `OS` was probably not intentional, was it?